go/fixchain/post.go

```diff
@@ -0,0 +1,59 @@
+package fixchain
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"github.com/google/certificate-transparency/go/x509"
+)
+
+// Post attempts to post the given chain to the Certificate Transparency log at
+// the given url, using the given http client.  Post returns a FixError if it
+// is unable to post the chain either because client.Post() failed, or the http
+// response code returned was not 200.  It is up to the caller to handle such
+// errors appropriately.
+func Post(chain []*x509.Certificate, client *http.Client, url string) *FixError {
```

Comment:
I actually quite liked the simplicity of it.  The asynchronous Logger contains a lot of functions containing some combination of the words 'post', 'chain', and 'log', and this being the base synchronous function that actually does do the post, it seemed fitting to give it a simple name.  Also, I liked the duality of Fix and Post being our two exported synchronous functions.  But I can call it  'PostChainToLog' or something similar if you really insist :P